docs/stdlib-times.md

```diff
@@ -72,6 +72,7 @@ times := import("times")
   2006" would be interpreted if it were the value; it serves as an example of
   the input format. The same interpretation will then be made to the input
   string.
+- `parseInLocation(format string, s string, loc string)`: parseInLocation is like parse but parse interprets a time as UTC; parseInLocation interprets the time as in the given location.
```

Comment:
IIRC `parse` will allow the timezone for interpreting the time in the format string. If so this seems redundant.

Reply:
True, but if I'm not mistaken a location can have a different timezone per Location based on the current time.  At my location the timezone now is UTC+2 (CEST) but at 01/01/2022 it is UTC+1 (CET).